app/assets/javascripts/chats.js.erb

```diff
@@ -23,8 +23,9 @@ $(document).ready(function(){
         scroll_bottom()
     })
     .fail(function( jqxhr, textStatus, error ) {
```

Comment:
I saw this repeated a lot, maybe you can move it to a method in the helper.js called errorHandling and then call it from here.
fail(errorHandling(jqxhr, textStatus, error))